src/test/java/integration/android/AndroidAppiumConditionsTest.java

```diff
@@ -0,0 +1,43 @@
+package integration.android;
+
+import com.codeborne.selenide.appium.AppiumCollectionCondition;
+import com.codeborne.selenide.appium.AppiumCondition;
+import com.codeborne.selenide.appium.SelenideAppium;
+import io.appium.java_client.AppiumBy;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+import static com.codeborne.selenide.WebDriverRunner.closeWebDriver;
+
+class AndroidAppiumConditionsTest extends BaseApiDemosTest {
+
+  @BeforeEach
+  void setUp() {
+    closeWebDriver();
+    SelenideAppium.launchApp();
+  }
+
+  @Test
+  void testAppiumCollectionConditionAttribute() {
```

Comment:
It's not needed to add `test` prefix to method names. Annotation `@Test` already does this job.

Reply:
Removed test prefix from method names